From 2ba9a29e0da6b83074a43c19e24c72b7ebde3aeb Mon Sep 17 00:00:00 2001
From: Ileana Dumitrescu <ileanadumitrescu95@gmail.com>
Date: Thu, 24 Jul 2025 18:16:44 +0300
Subject: [PATCH 2/2] tests: Remove CPPFLAGS from some compile commands

* tests/flags.at, tests/infer-tag.at: Remove unneeded CPPFLAGS from
  OBJC & OBJCXX compile commands.
---
 tests/flags.at     | 4 ++--
 tests/infer-tag.at | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/tests/flags.at b/tests/flags.at
index 82cfd8a6..d88eea02 100644
--- a/tests/flags.at
+++ b/tests/flags.at
@@ -58,7 +58,7 @@ int main() {
     return 0;
 }
 ]])
-compile="$OBJC $CPPFLAGS $OBJCFLAGS" link="$OBJC $OBJCFLAGS $LDFLAGS" source=a.m
+compile="$OBJC $OBJCFLAGS" link="$OBJC $OBJCFLAGS $LDFLAGS" source=a.m
 ],
 [OBJCXX], [AT_DATA([a.mm],
 [[#import <Foundation/Foundation.h>
@@ -81,7 +81,7 @@ int main() {
     return 0;
 }
 ]])
-compile="$OBJCXX $CPPFLAGS $OBJCXXFLAGS" link="$OBJCXX $OBJCXXFLAGS $LDFLAGS" source="a.mm -o a.o"
+compile="$OBJCXX $OBJCXXFLAGS" link="$OBJCXX $OBJCXXFLAGS $LDFLAGS" source="a.mm -o a.o"
 ],
 [F77], [AT_DATA([a.f],
 [[      program main
diff --git a/tests/infer-tag.at b/tests/infer-tag.at
index 2f1818e2..e74fe8ed 100644
--- a/tests/infer-tag.at
+++ b/tests/infer-tag.at
@@ -79,7 +79,7 @@ int main() {
 }
 ]])
 
-AT_CHECK([$LIBTOOL --mode=compile $OBJC $CPPFLAGS $OBJCFLAGS -c a.m],
+AT_CHECK([$LIBTOOL --mode=compile $OBJC $OBJCFLAGS -c a.m],
 	 [], [ignore], [ignore])
 
 AT_CLEANUP
@@ -113,7 +113,7 @@ int main() {
 }
 ]])
 
-AT_CHECK([$LIBTOOL --mode=compile $OBJCXX $CPPFLAGS $OBJCXXFLAGS -c a.mm -o a.o],
+AT_CHECK([$LIBTOOL --mode=compile $OBJCXX $OBJCXXFLAGS -c a.mm -o a.o],
 	 [], [ignore], [ignore])
 
 AT_CLEANUP
-- 
2.45.2

